cfgmgr/vrfmgr.cpp

```diff
@@ -207,6 +225,46 @@ void VrfMgr::doTask(Consumer &consumer)
         auto vrfName = kfvKey(t);
 
         string op = kfvOp(t);
+		// Mgmt VRF table event handling for in-band management
+        if (consumer.getTableName() == CFG_MGMT_VRF_CONFIG_TABLE_NAME) {
+            SWSS_LOG_DEBUG("Event for mgmt VRF op %s", op.c_str());
+            if (op == SET_COMMAND) {
+                bool in_band_mgmt_enabled = false;
+                bool mgmt_vrf_enabled = false;
+                for (auto i : kfvFieldsValues(t))
+                {
+                    if (fvField(i) == "mgmtVrfEnabled") {
+                        if (fvValue(i) == "true") {
+                            mgmt_vrf_enabled = true;
+                        } else {
+                            op = DEL_COMMAND;
```

Comment:
Why are we overwriting the `op` command here?. This operation is expected from the notification

Reply:
We should delete the VRF (Virtual router) from HW when no in-band-intf is part of the mgmt VRF i.e when in_band_mgmt_enabled is set false, we can set the op as DEL_COMMAND to delete the VRF.